eksctl/openscapes.jsonnet

```diff
@@ -92,12 +90,12 @@ local daskNodes = [
     ],
     nodeGroups: [n + {clusterName:: $.metadata.name} for n in [
         ng {
-            name: 'core-a',
+            name: 'core-b',
             availabilityZones: [nodeAz],
             ssh: {
                 publicKeyPath: 'ssh-keys/openscapes.key.pub'
             },
-            instanceType: "m5.xlarge",
+            instanceType: "r5.xlarge",
```

Comment:
May I ask why the core node size change too? Aren't r5 nodes more expensive than m5? Does it result in a reduction in total number of nodes? If not, I'd love for this to still be m5 to be consistent with other clusters.

Reply:
We have clusters with multiple core nodes, and I think its related to a constraint memory more than anything else, where prometheus-server is a big complexity biting us with its extreme memory footprint during startup. I've seen a cluster with 4 core nodes for example.

Maybe getting many core nodes relates to user pods scheduling on core nodes (#2163), I'm not sure.

I'm thinking quite confidently at the moment this is a cost saver due to prometheus-server making us memory constrained overall. The cost of a m5 vs r5 is $0.192 vs $0.252, a ratio of 1.3125 or invesively 0.7619. This is ~4:3 in CPU per $ and 4:8 in GB Ram per $ when comparing m5:r5.

I'll go for a merge here as this reflects the current state @yuvipanda! Do you think its okay to stick with r5 for the core node pool or should i change it back?

Reply:
@consideRatio I think it's ok for now! What I'd love is for us to evaluate what nodes actually make sense for core nodes and make that uniform :) The current setup again is just a guess.

Reply:
@consideRatio I've approved!